app/components/MachinesPage/MachinesQuoteForm.tsx

```diff
@@ -0,0 +1,129 @@
+// LIBRARY IMPORTS
+import {
+  Button,
+  chakra,
+  FormControl,
+  Heading,
+  HStack,
+  Input,
+  Textarea,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import React, { Dispatch, SetStateAction } from "react";
+import { useForm } from "react-hook-form";
+
+// LOCAL IMPORTS
+import { useSendQuoteRequest } from "@/utils/hooks/useSendQuoteRequest";
+import { PageViewMachinesBlocksQuoteSection } from "@/tina/__generated__/types";
+
+// TYPE DEFINTIONS
+interface MachineLayoutQuoteFormProps
+  extends PageViewMachinesBlocksQuoteSection {
+  setSubmitSuccessful: Dispatch<SetStateAction<boolean>>;
+}
+
+const ChakraForm = chakra("form");
+
+export default function MachinesQuoteForm(props: MachineLayoutQuoteFormProps) {
+  const { register, handleSubmit, errors, onSubmit } = useSendQuoteRequest();
+
+  return (
+    <ChakraForm
+      pt={14}
+      pb={24}
+      bg="brand.primary"
+      onSubmit={handleSubmit((data) =>
+        onSubmit(data, props.setSubmitSuccessful),
+      )}
+    >
+      <Heading textAlign="center" fontSize="3xl" mt={8} color="white">
+        {props?.requestQuoteForm?.formTitle || "Request a Quote"}
+      </Heading>
+      <VStack
+        spacing={6}
+        px={8}
+        py={10}
+        borderRadius="lg"
+        boxShadow="xl"
+        maxW="6xl"
+        mx="auto"
+      >
+        <HStack w="full" spacing={6}>
+          <FormControl isRequired>
+            <Input
```

Comment:
I think we can add height prop to input fields to make them a bit larger. They look quite small right now, what do you think?
Also, maybe add `fontSize="lg"` to get closer to wireframe?
![Screen Shot 2024-03-06 at 2 11 06 PM](https://github.com/wavecrestweb/kenworthy-machine/assets/109827834/618e725e-61aa-46a1-b986-b0a3b4c2a771)